src/pages/Script/index.tsx

```diff
@@ -0,0 +1,288 @@
+      content: scriptType,
+    },
+    {
+      title: i18n.t('scripts.type_id'),
+      content: typeId,
+    },
+    {
+      title: i18n.t('address.code_hash'),
+      content: (
+        <span>
+          {codeHash && `${isMobile() ? adaptMobileEllipsis(codeHash, 60) : adaptPCEllipsis(codeHash, 10, 80)}`}
+          <span
+            style={{
+              marginLeft: '1rem',
+            }}
+          >
+            <CopyOutlined
+              onClick={() => {
+                navigator.clipboard.writeText(codeHash).then(
+                  () => {
+                    dispatch({
+                      type: AppActions.ShowToastMessage,
+                      payload: {
+                        message: i18n.t('common.copied'),
+                      },
+                    })
+                  },
+                  error => {
+                    console.error(error)
+                  },
+                )
+              }}
+            />
+          </span>
+        </span>
+      ),
+    },
+    {
+      title: i18n.t('scripts.capacity_of_deployed_cells'),
+      content: <DecimalCapacity value={localeNumberString(shannonToCkb(capacityOfDeployedCells))} />,
+    },
+    {
+      title: '',
+      content: '',
+    },
+    {
+      title: i18n.t('scripts.capacity_of_referring_cells'),
+      content: <DecimalCapacity value={localeNumberString(shannonToCkb(capacityOfReferringCells))} />,
+    },
+  ]
+
+  return items
+}
+
+const ScriptsTitleOverview = ({ scriptInfo }: { scriptInfo: ScriptInfo }) => {
+  const dispatch = useDispatch()
+  return (
+    <div className={styles.scriptsTitleOverviewPanel}>
+      <OverviewCard items={getScriptInfo(scriptInfo, dispatch)} hideShadow />
+    </div>
+  )
+}
+
+const initScriptInfo: ScriptInfo = {
+  scriptName: '',
+  scriptType: '',
+  typeId: 0,
+  codeHash: '',
+  hashType: '',
+  capacityOfDeployedCells: '',
+  capacityOfReferringCells: '',
+  countOfTransactions: 0,
+  countOfDeployedCells: 0,
+  countOfReferringCells: 0,
+  pageOfTransactions: 1,
+  pageOfDeployedCells: 1,
+  pageOfReferringCells: 1,
+  sizeOfTransactions: 10,
+  sizeOfDeployedCells: 10,
+  sizeOfReferringCells: 10,
+}
+
+const seekScriptName = (codeHash: string, hashType: string): string =>
+  scriptHashNameMap.has(`${codeHash}_${hashType}`)
+    ? scriptHashNameMap.get(`${codeHash}_${hashType}`)!
+    : i18n.t('scripts.unnamed_script')
+
+export const ScriptPage = () => {
+  const history = useHistory()
+  const { search } = useLocation()
+  const { codeHash, hashType, tab } = useParams<{ codeHash: string; hashType: string; tab: ScriptTabType }>()
+  const anchor = (tab ?? 'transactions') as ScriptTabType
+  const [activeTab, setActiveTab] = useState<ScriptTabType>(anchor)
+  const parsed = queryString.parse(search)
+  const currentPage = parsePageNumber(parsed.page, PageParams.PageNo)
+  const pageSize = parsePageNumber(parsed.size, PageParams.PageSize)
+  initScriptInfo.codeHash = codeHash
```

Comment:
It's not recommended to update a variable outside the component, especially it's used as an initialized state.

The definition of `initScriptInfo` could be moved into the component

Reply:
> It's not recommended to update a variable outside the component, especially it's used as an initialized state.
> 
> The definition of `initScriptInfo` could be moved into the component

fixed